website/docs/d/cis_firewall.html.markdown

```diff
@@ -1,14 +1,15 @@
 ---
 layout: "ibm"
 page_title: "IBM: ibm_cis_firewall"
-sidebar_current: "docs-ibm-cis-firewall"
```

Comment:
it should be inline with datasource name.. please revert it

Reply:
reverted